frontend/views/containers/access/SignupForm.vue

```diff
@@ -71,7 +70,6 @@ export default ({
     validationsDebouncedMixins
   ],
   components: {
-    ModalTemplate,
```

Comment:
spotted an unused local component here, so removed it even though it's not related to this bug.